tools/mo/requirements.txt

```diff
@@ -1,6 +1,8 @@
-numpy>=1.16.6
-networkx~=2.5; python_version <= "3.6"
-networkx<=2.8.8; python_version > "3.6"
-defusedxml>=0.7.1
-importlib-metadata; python_version < "3.8" and sys_platform == 'win32'
-openvino-telemetry>=2022.1.0
\ No newline at end of file
+-c ../../src/bindings/python/constraints.txt
+numpy>=1.16.6,<1.25.0
+networkx
+requests
+fastjsonschema
```

Comment:
please remove `fastjsonschema` because this is optional module and install only in case the validated version

Reply:
I moved the version ('fastjsonschema~=2.15.1') into `constraints.txt`. If the version is specified (it was not changed in this PR) is this okay? Or should I remove it even if version is specified?